File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        // var count that will

        let count = 0

        for (let i = 0; i < nums.length; i++){ 
            if (nums[i] == nums[i+1]) {
                count++
                return true
            }
        } 
    return false;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        target = len(nums) // 2
        count = {}

        for num in nums:
            if num not in count:
                count[num] = 0
            count[num] += 1
        
        for num, count in count.items():
            if count > target:
                return num